### PR TITLE
fixed download link

### PR DESCRIPTION
**Pull request by easylogic**
This is to change main dowload link  to  https://github.com/summernote/summernote/releases/download/v0.8.18/summernote-0.8.18-dist.zip

### CODE REVIEW

**Comment by DiemenDesign** [comment]
Is there a download of master that we can reference so we don't have to update this link with new releases?

**Comment by easylogic** [comment]
@DiemenDesign   

No,  master branch is not exists currently. 

I think we should be update download link when a new release occurs .

**Review by DiemenDesign — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <h1 class="text-center index-title">
     <span class="img-index-title"></span>
     <span class="img-index-title-mobile">Super Simple WYSIWYG<br>Editor on Bootstrap</span>
-    <a href="{{ site.repository }}/archive/master.zip" class="index-download">
+    <a href="{{ site.repository }}/releases/download/v0.8.18/summernote-0.8.18-dist.zip" class="index-download">
```
Comment (hackerwins) [inline index.html:12]: I would like to change it as below.
```suggestion
    "{{ site.repository }}/releases/download/v{{ site.version }}/summernote-{{ site.version }}-dist.zip"
```

**Review by hackerwins — CHANGES_REQUESTED** [review]
Thanks for your contribution. I left a comment.

File: index.html (hunk 1 above)
Comment (easylogic) [reply index.html:12]: Good comment.

**Review by hackerwins — APPROVED** [review]
LGTM~ Thanks for contributing.

**Pull request merged**